### ISSUE TITLE
Switch to ESM

### ISSUE THREAD

**Issue by coderbyheart** [NordicSemiconductor/asset-tracker-cloud-aws-js#572]
~~AWS Lambda does not support ESM, yet. So this is on hold until it's implemented on AWS side.~~

ESM support in Node.js v14 is here: https://aws.amazon.com/de/blogs/compute/using-node-js-es-modules-and-top-level-await-in-aws-lambda/

**node-fetch v3**

- [x]  https://github.com/NordicSemiconductor/cloud-aws-firmware-ci-device-action/pull/264
- [ ] https://github.com/NordicSemiconductor/cloud-e2e-bdd-test-runner-js/pull/287
- [ ] https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js/pull/655
- [x] https://github.com/NordicSemiconductor/cloud-aws-package-layered-lambdas-js/pull/365 https://github.com/NordicSemiconductor/cloud-aws-package-layered-lambdas-js/pull/380

**Comment by coderbyheart** [comment]
Blocked by ESM lambda not working with layers.

**Comment by pfried** [comment]
Hi Markus, I guess you are also still stuck with this? I created an issue here: https://github.com/aws/aws-cdk/issues/23333

**Comment by coderbyheart** [comment]
A few days ago I started a new CDK TypeScript project and think we are now good to move to ESM: see #1185.

I have not observed the problem that you describe in your issue with having to maintain multiple tsconfig files.

**Comment by pfried** [comment]
Great! I will take a look
